docs/index.md

```diff
@@ -8,44 +8,50 @@
 <a id="the-challenge"></a>
 ## The challenge
 
-As an ML developer or data scientist, you typically want to write code in your preferred local development environment (IDE) or web notebook, and then run the same code on a larger cluster using scale-out containers or functions.
-When you determine that the code is ready, you or someone else needs to transfer the code to an automated ML workflow (for example, using [Kubeflow Pipelines](https://www.kubeflow.org/docs/pipelines/pipelines-quickstart/)).
-This pipeline should be secure and include capabilities such as logging and monitoring, as well as allow adjustments to relevant components and easy redeployment.
+Most data science solutions and platforms on the market today begin and therefore emphasize the research workflow. 
+When it comes time to integrate the generated models into real-world AI applications, they have significant functionality gaps.
 
-After you develop your model and feature engineering logic you need to deploy those into production pipelines 
-with real-time feature engineering, online model serving, API and data integrations, model and data quality 
-monitoring, no intrusive upgrades and so on. 
+These types of solutions tend to be useful only for the model development flow, and contribute very little to the production pipeline: 
+automated data collection and preparation, automated training and evaluation pipelines, real-time application pipelines, 
+data quality and model monitoring, feedback loops, etc.
 
-However, the implementation is challenging: various environments ("runtimes") use different configurations, parameters, and data sources.
-In addition, multiple frameworks and platforms are used to focus on different stages of the development life cycle.
-This leads to constant development and DevOps/MLOps work.
+To address the full ML application lifecycle, it’s common for organizations to combine many different tools which then forces 
+them to develop and maintain complex glue layers, or introduce manual processes, and creates technology silos that slow down 
```

Comment:
...or introduce manual processes, and thereby creating technology silos that slow down...

Reply:
manual process and silos are not the same, separate points, removed the `or`